lib/GeoExt/data/Print3Provider.js

```diff
@@ -0,0 +1,374 @@
+/**
+ * Copyright (c) 2008-2012 The Open Source Geospatial Foundation
+ *
+ * Published under the BSD license.
+ * See http://svn.geoext.org/core/trunk/geoext/license.txt for the full text
+ * of the license.
+ */
+
+/**
+ * @require GeoExt/data/PrintProviderBase.js
+ */
+
+/** api: (define)
+ *  module = GeoExt.data
+ *  class = PrintProvider
+ *  base_link = `Ext.util.Observable <http://dev.sencha.com/deploy/dev/docs/?class=Ext.util.Observable>`_
+ */
+Ext.namespace("GeoExt.data");
+
+/** api: example
+ *  Minimal code to print as much of the current map extent as possible as
+ *  soon as the print service capabilities are loaded, using the first layout
+ *  reported by the print service:
+ *
+ *  .. code-block:: javascript
+ *
+ *      var mapPanel = new GeoExt.MapPanel({
+ *          renderTo: "mappanel",
+ *          layers: [new OpenLayers.Layer.WMS("wms", "/geoserver/wms",
+ *              {layers: "topp:tasmania_state_boundaries"})],
+ *          center: [146.56, -41.56],
+ *          zoom: 7
+ *      });
+ *      var printProvider = new GeoExt.data.Print3Provider({
+ *          url: "/geoserver/pdf",
```

Comment:
the example here suggests that GeoServer supports MapFish v3, but is this really the case?